.gitignore

```diff
@@ -23,3 +24,4 @@ src/test/data/sandbox/
 docs/_site/
 *.class
 *.class
+*.class
```

Comment:
just curious what does this do?

Reply:
tells git to ignore all class files if im not wrong but i think i duplicated it